dosomething-dev/fastly-frontend/ashes_recv.vcl

```diff
@@ -27,8 +27,8 @@ else if (req.url.path ~ "(?i)\/((us|mx|br)\/)?(fact|sobre|volunteer|voluntario|r
   set req.http.X-Fastly-Backend = "ashes";
 }
 else if (req.url.path ~ "(?i)\/((us|mx|br)\/)?about/([A-Za-z0-9_\-]+)" &&
-    ! table.lookup(phoenix_about, std.tolower(re.group.3))) {
-  # About pages default to Ashes, but we'll opt some paths to Phoenix.
+    table.lookup(phoenix_about, std.tolower(re.group.3))) {
```

Comment:
I think this needs to be `ashes_about` to match the change in `ashes_init.vcl` 👾